### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by midopooler**
## Proposed changes

```
[FIX] Just fixed grammatical errors in README.md
```
## Issue(s)

N-A

## How to test or reproduce

NA

## Screenshots
NA

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [x] I have added necessary documentation (if applicable)
- [x] Any dependent changes have been merged and published in downstream modules

## Changelog

Just fixed grammatical errors in README.md

## Further comments

No such

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -206,15 +206,15 @@ Deploy Rocket.Chat stack to your server with [D2C](https://d2c.io/). Scale with
 [![Deploy](https://github.com/mastappl/images/blob/master/deployTo.png)](https://panel.d2c.io/?import=https://github.com/d2cio/rocketchat-stack/archive/master.zip/)
 
 ## Syncloud.org
-Run Rocket.Chat on your easy to use personal device.
+Run Rocket.Chat on your easy to use a personal device.
```
Comment (shivaylamba) [inline README.md:209]: This is wrong. The existing sentence is correct @midopooler

Comment (Sing-Li) [reply README.md:209]: can you fix the PR?

File: README.md [hunk 2]
```diff
@@ -114,11 +114,11 @@ Deploy to a DigitalOcean droplet with our one-click install listing from the Dig
 
 ## Layershift
 
-Instantly deploy your Rocket.Chat server for free on next generation auto-scaling PaaS.
+Instantly deploy your Rocket.Chat server for free on next-generation auto-scaling PaaS.
 
 [![Layershift Hosting](https://github.com/Sing-Li/bbug/raw/master/images/layershift.png)](http://jps.layershift.com/rocketchat/deploy.html)
 
-Painless SSL. Automatically scale your server cluster based on usage demand.
+Painless SSL. Automatically scale your server cluster based on user demand.
```
Comment (Sing-Li) [inline README.md:121]: drop `user` altogether.

Comment (midopooler) [reply README.md]: yes, done

**Comment by midopooler** [comment]
Resolved conflicts

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Closing this PR because it's stale now and it was already fixed in another one.

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there

**Pull request closed without merging**